sources/vega/commands/v1/validator_commands.proto

```diff
@@ -24,6 +24,8 @@ message NodeRegistration {
   string name = 7;
   // AvatarURL of the validator
   string avatar_url = 8;
+  // Number of Vega public key
+  uint32 vega_pub_key_number = 9;
```

Comment:
This might need an update in the genesis block next time we restart the network right? Should we log that somewhere / at least on the core in the CHANGELOG. cc @gordsport

Reply:
As in for new network it would be ideal. Although for old nodes it is backwards compatible as the key number they are using is assumed to be 0.

Reply:
ah yea true that.